### ISSUE TITLE
go install error

### ISSUE THREAD

**Issue by redonkulus** [rhysd/changelog-from-release#13]
I'm pretty new to Go so please excuse me if I am doing this wrong. Following the instructions to install the package gives me version `v2.2.2`. I read that @latest does not work, when trying to specify a version, that does not work either:

```shell
# does not work since I'm installing this in a blank directory
$ go install github.com/rhysd/changelog-from-release
go: 'go install' requires a version when current directory is not in a module
	Try 'go install github.com/rhysd/changelog-from-release@latest' to install the latest version

# using @latest does work (even though README says it does not)
$ go install github.com/rhysd/changelog-from-release@latest
go: finding module for package golang.org/x/oauth2
go: finding module for package github.com/google/go-github/github
go: finding module for package github.com/pkg/errors
go: found github.com/google/go-github/github in github.com/google/go-github v17.0.0+incompatible
go: found github.com/pkg/errors in github.com/pkg/errors v0.9.1
go: found golang.org/x/oauth2 in golang.org/x/oauth2 v0.0.0-20220808172628-8227340efae7
go: finding module for package github.com/google/go-querystring/query
go: found github.com/google/go-querystring/query in github.com/google/go-querystring v1.1.0

# confirm old version
$ ./go/bin/changelog-from-release -v
v2.2.2

# specifying version does not work
$ go install github.com/rhysd/changelog-from-release@v3.1.4
go: github.com/rhysd/changelog-from-release@v3.1.4: github.com/rhysd/changelog-from-release@v3.1.4: invalid version: module contains a go.mod file, so module path must match major version ("github.com/rhysd/changelog-from-release/v3")
```

```shell
$ go version
go version go1.19 darwin/amd64
```

**Comment by rhysd** [comment]
Thanks. I also could reproduce this. It seems that Go module's behavior changed. I'll update readme doc.

**Comment by redonkulus** [comment]
@rhysd is there a reason that latest does not pull in the newest version?

**Comment by rhysd** [comment]
It didn't work previously, but it works now.

In older version of Go, `go install github.com/rhysd/changelog-from-release@latest` caused an error because this project reached v2. Go required `go install github.com/rhysd/changelog-from-release/v2@latest` even if building an executable at that time.

**Comment by rhysd** [comment]
If `go install` installs older version, your local copy may be old. I think `go get -u` can update it.

**Comment by rhysd** [comment]
I fixed this issue at v3.2.0. The following command should work (notice `/v3` semantic path)

```
go install github.com/rhysd/changelog-from-release/v3@latest
```

**Comment by redonkulus** [comment]
Ok thanks for fixing it and validating it!
